identity-android/src/main/java/com/android/identity/android/mdoc/deviceretrieval/VerificationHelper.java

```diff
@@ -81,15 +85,8 @@
 // cleaned up at object finalization time.
 @SuppressWarnings("NotCloseable")
 public class VerificationHelper {
-
     private static final String TAG = "VerificationHelper";
 
```

Comment:
these now live in `com.android.identity.util.EngagementTypeDef` in the **identity** module and can now be referenced in all modules. Previously these values were not accessible to the **identity** module (only **identity-android** module)